aten/src/ATen/WrapDimUtils.h

```diff
@@ -72,4 +72,11 @@ static inline int64_t legacy_cat_wrap_dim(int64_t dim, TensorList tensors) {
   return dim;
 }
 
+// wrap negative dims in to_transform_dims
```

Comment:
what is to_transform_dims?

Reply:
oops, I should probably change it to "wrap negative dims in a vector"